pkg/kubelet/dockertools/manager.go

```diff
@@ -486,6 +487,15 @@ func makeEnvList(envs []kubecontainer.EnvVar) (result []string) {
 	return
 }
 
+func makeEnvMap(envs []kubecontainer.EnvVar) map[string]string {
```

Comment:
thoughts about moving this into

https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/kubelet/container/helpers.go

```
func EnvVarsToMap(...) ... {
}
```